addons/mail/static/src/components/discuss_sidebar_item/discuss_sidebar_item.js

```diff
@@ -117,6 +118,10 @@ class DiscussSidebarItem extends Component {
      * @param {MouseEvent} ev
      */
     _onClick(ev) {
+        if (isEventHandled(ev, 'EditableText.click')) {
+            // handled in `_onClickedEditableText`
```

Comment:
Not very important but comment is overkill, we know where it comes from with the event handled name.